src/Versioning/Provider.php

```diff
@@ -0,0 +1,8 @@
+<?php declare(strict_types = 1);
+
+namespace Pd\Forms\Versioning;
+
+interface Provider
+{
+	public function generatePathWithVersion(string $filePath): string;
```

Comment:
Nedáme do balíčku nějakou default implementaci? Jakože vrátí to jen filepath bez úpravy. Čistě kvůli tomu aby to fungovalo by default. Teda za předpokladu že to potom půjde rozumně nahradit projektově :)